intro/classes/index.html

```diff
@@ -185,19 +152,165 @@ <h3>Single leading underscores</h3>
+{% endhighlight %}
+
+            Note that I've put a few more arguments into the constructor: <code>position</code>, <code>image_path</code>, and 
+            <code>size</code>. These are bits of information that I would expect pretty much any character in the game to need.
+            Now, I want to create an NPC class. I <i>could</i> rewrite all of that code in the constructor of
+            my NPC class, but that's a lot of overhead as mentioned before. Instead, let's <b>subclass</b> <code>Actor</code>:
+
+{% highlight python %}
+class NPC(Actor):
+    def __init__(self, position, image_path, size, dialogue):
+            self.dialogue = dialogue
+{% endhighlight %}
+            
+            And that's all there is to it right? <b>Wrong.</b>
+        </p>
+        
+        <p>
+            Python has no idea what our intentions are. It could assume that we want the constructor of <code>Actor</code> to be
+            called immediately, but that's a lot of overhead and might not actually be what we want. So, we have to explicitly
+            call the constructor of <code>Actor</code> (side note: Actor here is also known as the superclass of NPC, and that'll
+            be important in a second). There are a couple of ways to do that: explicitly calling the constructor of <code>Actor</code>,
+            or by calling the constructor of the superclass.
+        </p>
+
+        <p>
+            Calling Actor's constructor:
+
+{% highlight python %}
+class NPC(Actor):
+    def __init__(self, position, image_path, size, dialogue):
+            Actor.__init__(self, position, image_path, size)
+            self.dialogue = dialogue
+{% endhighlight %}
+
+            Calling the superclass constructor:
+
+{% highlight python %}
+class NPC(Actor):
+    def __init__(self, position, image_path, size, dialogue):
+            super().__init__(position, image_path, size)
+            self.dialogue = dialogue
+{% endhighlight %} 
+        </p>
+        
+        <p>
+            These look practically identical (and in fact, they practically do the exact same thing in simple cases), but there are a
+            few reasons to choose one over the other. Generally, <code>super().__init__</code> is preferred because it's easy to swap
+            out or rename the superclass if you need to, as well as not needing to remember the <code>self</code> parameter. And for
+            longer class names, it's also shorter. But, the situation where you would possibly want to use <code>Actor.__init__</code>
+            would be in the case of more complicated inheritance structures (long chains of inheritance or multiple inheritance,
+            both of which are well outside the scope of this article and game design in general, but a link is at the bottom of the
+            page for an excellent video on the topic from the 
+            <a href="https://www.youtube.com/c/mCodingWithJamesMurphy" class="new-tab">mCoding</a> YouTube channel).
```

Comment:
Link to channel here but to video at the bottom? Is it so that they don't immediately go watch that video, makes sense then